its/src/test/java/com/sonar/it/scanner/msbuild/utils/BuildCommand.java

```diff
@@ -41,7 +41,7 @@ public class BuildCommand extends BaseCommand<BuildCommand> {
 
   private final ArrayList<String> arguments = new ArrayList<>();
   private String dotnetCommand;
-  private boolean explicitRestore;
+  private boolean invokeNuGetRestore;
```

Comment:
This does not  indicate a yes/no question. It's more like it does an action.
Please go for "ShouldInvokeNugetRestore" or something that indicates yes/no answer.